### PR TITLE
Release/5.0

### LINKED ISSUE #12: Allow a data bucket to be in a different region to the regular bucket

### LINKED ISSUE #19: Gracefully handle running ls with no data-bucket configured

### LINKED ISSUE #20: Review the naming of CLI options around anonymised data exports
The `--data-bucket` option is quite confusing, and doesn't describe what it does very well.

### LINKED ISSUE #21: Refactor YML variable names to remove hyphens
The YML standard does not like hyphens in variable names, I have pulled the config file into ansible which accepts YML files and it fails. We need to refactor the hyphens to underscores.

I understand the code works with them with hyphens right now, so perhaps this change could be made in a non-breaking way? In other words, it would continue to understand the variable names with hyphens, but also with underscores, and if the configure command is run it would update/set them to be underscores. 

Let me know what you think that is necessary.

### LINKED ISSUE #56: Magedbm creating yaml config with curly braces
When using `magedbm2 configure` the `~/.magedb.2/config.yml` is created with curly braces in it which isn't valid yaml.

Screenshot 2020-08-10 at 09 47 28

### PR DESCRIPTION
**Pull request by vlad1992**
This is just the release containing recent PR's.
Closes #12 
Closes #19
Closes #20 
Closes #21 
Closes #56

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ All notable changes will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.0.0] - 2022-04-01
+### Changed
+- hyphens are no longer allowed in variable names in yaml files
```
Comment (tgerulaitis) [inline CHANGELOG.md:9]: Could we highlight this as a backwards incompatible change? Maybe with a `**breaking**` in front?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,15 @@ All notable changes will be documented in this file.
...
+- an empty config file can no longer be created via the configure method
+- data bucket config name modified to reflect the fact that it denotes the bucket where the anonymised exports will be kept
```
Comment (tgerulaitis) [inline CHANGELOG.md:11]: Could we mark this as a backwards incompatible change?

**Review by tgerulaitis — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (vlad1992) [reply CHANGELOG.md:11]: I can add that, I just thought that making it a major version upgrade would do that so that's why I didn't add it initially

Comment (tgerulaitis) [reply CHANGELOG.md]: Yeah, it tells you that the version contains breaking changes, but it's nice to tell the users which of the changes are breaking, so they don't have to figure out themselves ;)

**Pull request merged**